exercises/concept/lasagna-master/.docs/instructions.md

```diff
@@ -49,8 +49,7 @@ It was amazing and had something special to it.
 The friend sent you the list of ingredients and told you the last item on the list is the "secret ingredient" that made the meal so special.
 Now you want to add that secret ingredient to your recipe as well.
 
-Write a function `addSecretIngredient` that accepts two `string` vectors of ingredients as parameters.
-The first parameter is the list your friend sent you, the second is the ingredient list of your own recipe.
+Write a function `addSecretIngredient` that accepts a reference to a `string` of vectors representing the ingredient list of your own recipe and a `string` of vectors that is the list your friend sent you.
```

Comment:
I prefer the two sentence version. It should be a `vector of strings`.
How about we change line 53 to `The first parameter is a reference to the list your friend ...`?

Reply:
Thanks @vaeng.
With [this commit](8db1be52d957df04c07cde78f5ee97681d1d8d83) I tried to address your suggestion.